<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MF_trailerDecoderDialog</class>
 <widget class="QDialog" name="MF_trailerDecoderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Trailer Data:(Input the Access Bits, like &quot;FF 07 80 69&quot; or &quot;FF 07 80&quot;)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="accessBitsEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="isAccessBitsValidLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="sizeGroupBox">
       <property name="title">
        <string>Blocks</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="size4Button">
          <property name="text">
           <string>4</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="size16Button">
          <property name="text">
           <string>16</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Data Block Permission:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="dataBlockWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>4</verstretch>
      </sizepolicy>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <row>
      <property name="text">
       <string>Block0</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Block1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Block2</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Read</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Write</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Increase</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Decrease/Transfer/Restore</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Trailer Block Permission:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="trailerBlockWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <row>
      <property name="text">
       <string>Read</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Write</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>KeyA</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Access Bits</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>KeyB</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Reference:
MF1S70YYX_V1 Product data sheet
Rev. 3.2 â€” 23 November 2017</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MF_trailerDecoderDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MF_trailerDecoderDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
